topics.ts

```diff
@@ -1,6 +1,6 @@
-type IndexedTopics = Record<string, string>;
+type IndexedTopicsType = Record<string, string>;
```

Comment:
Made this change to resolve ts lint error.
`'IndexedTopics' is already defined  @typescript-eslint/no-redeclare`